spinnaker_camera_driver/src/camera.cpp

```diff
@@ -256,6 +256,12 @@ void Camera::setImageControlFormats(const spinnaker_camera_driver::SpinnakerConf
 
   // Set Pixel Format
   setProperty(node_map_, "PixelFormat", config.image_format_color_coding);
+
+  if (config.image_format_color_coding.find("Bayer") != std::string::npos)
+  {
+      ROS_INFO_STREAM("Bayer image format selected. Disabling ISP to maximize framerate.");
+      setProperty(node_map_, "IspEnable", false);
```

Comment:
this is the thing that helped frame rates right?
curious why isp is a slowdown anyway.
but make sure a comment about this shows up in your commit message (it should probably be a separate commit if it doesn't affect ptp timestamping cleanup directly for some reason).